routes/users/view.js

```diff
@@ -1,14 +1,219 @@
 const debug = require('debug');
 
 const api = require('../../lib/api');
+const orcid = require('../../lib/orcid');
 
-module.exports = (req, res) => {
-  const orcid = req.params.orcid;
+// Fetch user educations from orcid
+const getUserEducations = (orcidId) => new Promise((resolve) => {
+  return orcid.getPersonEducations(orcidId, (educationsErr, educations) => {
+    const allEducations = [];
 
-  debug('octopus:ui:debug')(`Showing User Profile: ${orcid}`);
+    // Extract specific data from orcid
+    if (educations && educations['affiliation-group']) {
+      educations['affiliation-group'].forEach((affiliation) => {
+        const summaries = affiliation.summaries;
 
-  return api.getUserByORCiD(orcid, (userErr, userData) => {
-    res.locals.person = userData;
+        summaries.forEach((summary) => {
+          const educationSummary = summary['education-summary'];
+          const organizationName = educationSummary.organization.name;
+          const roleTitle = educationSummary['role-title'];
+          const educationsGroup = { organizationName, roleTitle };
+
+          allEducations.push(educationsGroup);
+        });
+      });
+    }
+
+    // Return it
+    resolve(allEducations);
+  });
+});
+
+// Fetch user employments from orcid
+const getUserEmployments = (orcidId) => new Promise((resolve) => {
+  return orcid.getPersonEmployments(
+    orcidId,
+    (employmentsErr, employments) => {
+      const currentEmploymentArr = [];
+      const pastEmployments = [];
+
+      // Extract specific data from orcid
+      if (employments && employments['affiliation-group']) {
+        employments['affiliation-group'].forEach((affiliation, index) => {
+          const summaries = affiliation.summaries;
+
+          summaries.forEach((summary) => {
+            const employmentSummary = summary['employment-summary'];
+            const organizationName = employmentSummary.organization.name;
```

Comment:
Can we please use Lodash for all these value lookups?
For example: `_.get(summary, 'employmentSummary.organization.name')`

This will ensure that when the expected data is not present the app doesn't crash with `Cannot read "name" of undefined`.